### PR TITLE
Feature/support standardlogicapps

### PR DESCRIPTION
**Pull request by gcmlpfaltaher**
- Add support for diagnostic logs from Standard Logic Apps
    - Standard Logic Apps treat workflow failed runs as "Informational" messages, update to log appropriately in Sumo
- Deploy to nonprod tenants (gcmdev/gcmtst)
    - Update naming prefixes to include gcmdev/gcmtst for nonprod tenants

### CODE REVIEW

File: EventHubs/src/SumoEventHub.csproj [hunk 1]
```diff
@@ -6,19 +6,12 @@
     <PublishOutputPathContents>true</PublishOutputPathContents>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Gcm.AzureRM.PowerShell.Deployment" Version="1.*" />
-    <PackageReference Include="Gcm.ConfigurationLoader" Version="3.0.0-netstandard0039" />
+    <PackageReference Include="Gcm.AzureAz.PowerShell" Version="0.*" />
```
Comment (swettstein) [inline EventHubs/src/SumoEventHub.csproj]: is this in the trello board? if so, update it!

Comment (gcmlpfaltaher) [reply EventHubs/src/SumoEventHub.csproj]: Assumed it was earlier and upgraded without checking the board, but this repo does not show up in the trello board.  should i add it? and do you mean update to 1.* version for AzureAz?

Comment (swettstein) [reply EventHubs/src/SumoEventHub.csproj]: yeah add it. might as well use 1.* too

**Review by ltobiasg — APPROVED** [review]

**Review by ltrochmann-gcmlp — APPROVED** [review]

**Review by swettstein — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 2.3.0
+- Add support for diagnostic logs from Standard Logic Apps
+    - Standard Logic Apps treat workflow failed runs as "Informational" messages, update to log appropriately in Sumo
+- Update naming prefixes to include gcmdev/tst for nonprod tenants
+
```
Comment (haimizrael) [inline CHANGELOG.md:6]: Should add a note:
- Removed unnecessary references to Gcm.ConfigurationLoader

File: EventHubs/src/Deploy.parameters.ps1 [hunk 3]
```diff
@@ -1,7 +1,24 @@
-param ([Parameter(Mandatory = $true)][string]$Lifecycle, [string] $LifecyclePrefix)
+param([string] $Lifecycle, [string] $LifecycleSuffix)
 
-[System.Reflection.Assembly]::LoadFrom("$PSScriptRoot\Gcm.ConfigurationLoader.dll") | Out-Null
+Connect-AzureAccount | Out-Null
 
-$obj = [Gcm.ConfigurationLoader.ConfigurationLoader]::LoadHashtable("$PSScriptRoot/Config/deploy.app.json", "$PSScriptRoot/Config/deploy.variables.json", [string[]]@("default", $Lifecycle))
+# Fetch the App Insights Instrumentation Key programmatically for each environment
```
Comment (haimizrael) [inline EventHubs/src/Deploy.parameters.ps1:5]: Add add to the comment: 
At the time of writing, for the gcmdev/gcmtst tenants, the app insights instances are still using gcm- and not gcmdev-/gcmtst- as tenant prefixes.

**Review by haimizrael — COMMENTED** [review]
Looks good: just a couple of comments suggestions

**Pull request merged**